PF/Assignment2/assignment_1.java

```diff
@@ -0,0 +1,72 @@
+public class assignment_1 
+{
+public static void main(String args[])
+   {
+	assignment_1 as1=new assignment_1();
+	int row=9,n=5;
+	String pyramid[]=new String[row];
+	pyramid=as1.printPyramid(row, n);
+	for(int i=0;i<row;i++)
+	{
+		System.out.print(pyramid[i]);
+	    System.out.println();
+	}
+	
+   }
+String spaces(int row,int n)
+    {
+	  int k=row+1;
+	  String space="";
+	  if(k<=4)
+	  {
+	  for(int i=0;i<5-k;i++)
+		  space=space+" ";
+	  }
+	  else
+	  {
+		  for(int i=0;i<k-5;i++)			
+		  {
+			  space=space+" ";
+		  }
+	  }
+	  return space;
+	}
+String numbers(int row,int n)
```

Comment:
ACTION: the argument 'n' is nowhere used in the function. Looking at the code I think it does not work for 'n' rather it will work if n<= 5. You need to rewrite it so that it can use 'n'.